### ISSUE TITLE
Debug logging too long for python logging to accept

### ISSUE THREAD

**Issue by peterbe** [mariocesar/sorl-thumbnail#402]
I don't know what the limit on `msg` is on `self.socket.sendto(msg, self.address)` in `lib/python2.7/logging/handlers.py`. But I noticed that in Python 2.7.9 when I now run my tests I get a lot of messages like this:

```
Traceback (most recent call last):
  File "/opt/boxen/homebrew/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/handlers.py", line 868, in emit
    self.socket.sendto(msg, self.address)
error: [Errno 40] Message too long
Logged from file base.py, line 154
```

So I put in a piece of code there in `handlers.py` that would write all `msg` lines that are longer than 200 characters to a file. After some investigations I found that I got 12 of those logging warnings. And the lengths of the 12 longest `msg` strings are:
```
>>> for i in range(12):
...   print len(messages[i])
...
15715
9966
9966
9966
9966
9965
9296
9294
9294
9294
9293
9293
>>> len(messages[12])  # the 13th
6186
```
So basically, the limit seems to be anything more than ~6186. 
For example, here's a message: https://gist.github.com/peterbe/0b15d2fcb8348642fba0

Basically, in https://github.com/mariocesar/sorl-thumbnail/blob/1a9d719ed39333face547337dc7d4e0583e70090/sorl/thumbnail/base.py#L153-L154 we're allowing some binary things be included and that makes Python unhappy.
